src/qt/guiutil.cpp

```diff
@@ -18,6 +18,10 @@
 #include "script/standard.h"
 #include "util.h"
 
+#ifdef ENABLE_WALLET
```

Comment:
Is it possible to have Qt without the wallet?

Reply:
Yes. It's pretty!